### ISSUE TITLE
Standardize input values for size prop

### ISSUE THREAD

**Issue by msankaran0712** [trimble-oss/modus-web-components#658]
Any component should have standard input values for the `size` prop.

Should that be 'small', 'medium', and 'large' ?
or 
just 'small' and 'large' ? 
'medium' is default size so it is not required

@ryan-henness-trimble @coliff

**Comment by coliff** [comment]
yes, agreed. we should try and use small instead of condensed where possible. If class condensed exists already maybe it could be deprecated and available as an alias for small.

Note; not all components are available in all sizes though.

**Comment by msankaran0712** [comment]
> yes, agreed. we should try and use small instead of condensed where possible. If class condensed exists already maybe it could be deprecated and available as an alias for small.
> 
> Note; not all components are available in all sizes though.

Yes agreed.

**Comment by cjwinsor** [comment]
Review where we stand today, list out all variations here to determine next steps.

**Comment by mitch-trimble** [comment]
Hello world

**Comment by enowak1031** [comment]
@prashanthr6383  @mitch-trimble Let's review this one in the Three Amigos meeting.

**Comment by prashanthr6383** [comment]
> @prashanthr6383 @mitch-trimble Let's review this one in the Three Amigos meeting.

@prashanth-offcl please check
